Mollier/SAM.Core.Mollier.UI/Modify/AddMollierProcesses.cs

```diff
@@ -236,60 +268,70 @@ private static Series createProcessPointsSeries(this Chart chart, MollierPoint m
         
         
         // General methods used above 
-        private static List<UIMollierProcess> generateLabelsForProcessesSystems(List<List<UIMollierProcess>> systems)
+        private static List<UIMollierProcess> generateLabelsForProcessesSystems(List<IMollierGroup> groups)
```

Comment:
Ramane to generateLabels

ProcessesSystems not needed <- input points out what kind of object we are dealing with